Content.Server/Botany/Seed.cs

```diff
@@ -291,9 +291,6 @@ public IEnumerable<EntityUid> Harvest(EntityUid user, int yieldMod = 1)
         {
             AddToDatabase();
 
-            if (user == null)
-                return Enumerable.Empty<EntityUid>();
-
```

Comment:
whats this about?

Reply:
Not sure. The function was just doing a null-check on a non-nullable argument. I assume something funny happened during IEntity removal. Came up when looking for warnings about unnecessary null-checks.